src/components/Stats/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css";
+
+// eslint-disable-next-line react/prop-types
+const render = ({ label, value }) => (
+  <li className={styles.item}>
+    <p className={styles.label}>{label}</p>
+    <p className={styles.value}>{value}</p>
+  </li>
+);
+
+const Stats = ({ list }) => <ul className={styles.root}>{list.map(render)}</ul>;
```

Comment:
Why not inline this everything in the component?

Reply:
I could, but I hate nesting 😄

Reply:
BTW, we didn't had components for the label and value thingies right?

Reply:
I just think we should avoid disabling linter stuff, anything that allows for that comment to disappear is good to me.

```
const renderItem = (item) => {
   const {label, value} = item;

 return <li className={styles.item}>
    <p className={styles.label}>{label}</p>
    <p className={styles.value}>{value}</p>
  </li>
};
```

Also I think it's more clear a regular function like the one above instead of a "species of" react component like thingy. 

This makes it more clear that it's an helper render function, I believe

Reply:
Fair enough, I'll do the changes 👍 What about my questions about the components?

Reply:
Updated

Reply:
We had components for the labels I think :grimacing: , not the values though

Reply:
true, updated